ci/install.sh

```diff
@@ -117,130 +49,72 @@ build_gpg_beta() {
-  # alternatively, forcing the libgpg-error build to use gcc should work
-  export LD_LIBRARY_PATH="/usr/local/clang/lib"
-
-  git clone git://git.gnupg.org/npth
-  pushd npth
-  git checkout "$NPTH_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" --disable-doc
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/libgpg-error
-  pushd libgpg-error
-  git checkout "$LIBGPG_ERROR_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" --disable-doc
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/libgcrypt
-  pushd libgcrypt
-  git checkout "$LIBGCRYPT_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" --disable-doc --with-libgpg-error-prefix="${GPG_INSTALL}"
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/libassuan
-  pushd libassuan
-  git checkout "$LIBASSUAN_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" --disable-doc --with-libgpg-error-prefix="${GPG_INSTALL}"
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/libksba
-  pushd libksba
-  git checkout "$LIBKSBA_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" --disable-doc --with-libgpg-error-prefix="${GPG_INSTALL}"
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/pinentry.git
-  pushd pinentry
-  git checkout "$PINENTRY_VERSION"
-  cat << 'END' | git apply -
-diff --git a/Makefile.am b/Makefile.am
-index 8c8b8e5..412244c 100644
---- a/Makefile.am
-+++ b/Makefile.am
-@@ -85,7 +85,7 @@ endif
- SUBDIRS = m4 secmem pinentry ${pinentry_curses} ${pinentry_tty} \
- 	${pinentry_emacs} ${pinentry_gtk_2} ${pinentry_gnome_3} \
- 	${pinentry_qt} ${pinentry_tqt} ${pinentry_w32} \
--	${pinentry_fltk} ${pinentry_efl} doc
-+	${pinentry_fltk} ${pinentry_efl}
- 
- 
- install-exec-local:
-END
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" \
-    --with-libgpg-error-prefix="${GPG_INSTALL}" \
-    --with-libassuan-prefix="${GPG_INSTALL}" \
-    --enable-pinentry-curses \
-    --disable-pinentry-emacs \
-    --disable-pinentry-gtk2 \
-    --disable-pinentry-gnome3 \
-    --disable-pinentry-qt \
-    --disable-pinentry-qt5 \
-    --disable-pinentry-tqt \
-    --disable-pinentry-fltk
-  ${MAKE} -j${MAKE_PARALLEL} install
-  popd
-
-  git clone git://git.gnupg.org/gnupg.git
-  pushd gnupg
-  git checkout "$GNUPG_VERSION"
-  ./autogen.sh
-  ./configure --prefix="${GPG_INSTALL}" \
-    --with-libgpg-error-prefix="${GPG_INSTALL}" \
-    --with-libgcrypt-prefix="${GPG_INSTALL}" \
-    --with-libassuan-prefix="${GPG_INSTALL}" \
-    --with-ksba-prefix="${GPG_INSTALL}" \
-    --with-npth-prefix="${GPG_INSTALL}" \
-    --disable-ldap \
-    --disable-doc \
-    --enable-maintainer-mode
-  ${MAKE} -j${MAKE_PARALLEL} install
+  gpg_build="$PWD"
+  gpg_install="$GPG_INSTALL"
+  mkdir -p "$gpg_build" "${gpg_install}"
+  git clone --depth 1 https://github.com/riboseinc/gpg-build-scripts
```

Comment:
@dewyatt should we move this to the rnpgp org?

Reply:
@ronaldtse Probably, it certainly wouldn't hurt.

Reply:
Transferred.